api/integration_alert_channels_datadog.go

```diff
@@ -0,0 +1,123 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+type datadogSite string
+
+type datadogService string
+
+const (
+	//The list of valid inputs for DatadogService field
+	DatadogSiteEu  datadogSite = "eu"
+	DatadogSiteCom datadogSite = "com"
+
+	//The list of valid inputs for DatadogService field
+	DatadogServiceLogsDetails   datadogService = "Logs Detail"
+	DatadogServiceEventsSummary datadogService = "Events Summary"
+	DatadogServiceLogsSummary   datadogService = "Logs Summary"
+)
+
+// NewDatadogAlertChannel returns an instance of DatadogAlertChannel
+// with the provided name and data.
+//
+// Basic usage: Initialize a new DatadogAlertChannel struct, then
+//              use the new instance to do CRUD operations
+//
+//   client, err := api.NewClient("account")
+//   if err != nil {
+//     return err
+//   }
+//
+// datadog := api.NewDatadogAlertChannel("foo",
+//   api.DatadogChannelData{
+// 	  DatadogSite:    api.DatadogSiteEu.String(),
```

Comment:
Is this correct? 🤔 Shouldn't we just pass the `datadogSite` type as is?
```suggestion
// 	  DatadogSite:    api.DatadogSiteEu,
```